### DISCUSSION TITLE
MDX now has a dedicated #mdx channel on the NextJS Discord

### DISCUSSION THREAD

**Discussion by xHomu** [mdx-js/mdx#1919] [General]
Something us MDX fans been petitioning for a while now!

![image](https://user-images.githubusercontent.com/84349818/151256627-8b660be4-1972-4262-9e0a-809a7090055e.png)

http://nextjs.org/discord

Hopefully this will help concentrate solutions of next-specific questions in one place.

**Comment by PabloLION** [comment]
I'd like to know if there's any reason not to make a new discord server?

**Reply by PabloLION** [reply]
btw this channel is archived

**Reply by wooorm** [reply]
https://github.com/unifiedjs/unified/discussions/127
